### ISSUE TITLE
The twitter widget ID is not used any more

### ISSUE THREAD

**Issue by aht61** [lemberg/connfa-webapp#7]
Twitter does not use/give widget IDs any more, so the social media link does not fetch the twitter posts any more. Instead of the widget ID, a simple embed code (html) supplied by twitter could be pasted in the event backhand.

**Comment by brittyazel** [comment]
Yep, that's what we ended up doing here https://gitlab.gnome.org/Teams/Engagement/Events/connfa/connfa-webapp
